frontend/src/lib/components/Support/supportLogic.ts

```diff
@@ -71,7 +94,7 @@ export const supportLogic = kea<supportLogicType>([
         openSupportForm: async ({ kind, target_area }) => {
             actions.resetSendSupportRequest({
                 kind: kind ? kind : undefined,
-                target_area: target_area ? target_area : undefined,
+                target_area: target_area ? target_area : getURLPathToTargetArea(window.location.pathname),
```

Comment:
Should be able to just do:

```suggestion
                target_area: target_area ?? getURLPathToTargetArea(window.location.pathname),
```

and same for `kind` above.